meetings/2017-11-02.md

```diff
@@ -0,0 +1,63 @@
+# Security WG meeting 2017-11-02
+
+GitHub issue: https://github.com/nodejs/security-wg/issues/63
+Meeting video: https://youtu.be/UWKC1GTPAFo
+Previous meeting: https://github.com/nodejs/security-wg/blob/master/meetings/2017-10-12.md
+
+
+# Present
+
+- Bryan English (@bengl)
+- Sam Roberts (@sam-github)
+- Vladimir de Turckheim (@vdeturckheim)
+- Michael Alexander (@mgalexander)
+- Michael Dawson (@mhdawson)
+- Reed Loden (@reedloden)
+- Colin Ihrig (@cjihrig)
+- Arunesh Chandra (@aruneshchandra)-observer
+- Hitesh Kanwathirtha (@digitalinfinity)-observer
+- Adam Baldwin - late (@evilpacket)
+
+# Review of last meeting
+
+
+# New agenda
+
+## Hacker one setup/demo
+
+* @vdeturckheim gave us a walkthrough of what he has setup.
+* Easy to invite people to collaborate on specific vulnerabilities
+*
+
+##  Bug Bounty Program https://github.com/nodejs/security-wg/issues/49
+
+* want to add bounties Node.js to list they provide bounties for.
+* not a huge amount of money ($1500 at the top end)
+* does provide some additional motivation for researchers to find/report
+  Issues.
+* There is no cost to the project (Internet bug bounty would provide for free)
+* Only for security vulnerabilities, do they need to go through hacker one.
+* Needs to meet some minimum bar.
+* Michael: should probably get hacker one setup done first ?
+* Michael Alexander volunteered to champion this on the Security-wg, like with an
+  end of year timeframe.
+
+## Proposing an Early Disclosure Program  https://github.com/nodejs/security-wg/issues/58
+
+* One thing discussed was to get more input from the potential consumers
+* Vladimir will put together and issue, twitter etc to try to gather that input.
+
+## acknowledge privacy of info in private repos.https://github.com/nodejs/security-wg/issues/64
+
+* general consensus that having document and having people acknowledge in issue 64 is good
+  enough
+
+# Actions
+
+* @vdeturckheim to set up meeting with Core triage team to handover HackerOne Organization (https://github.com/nodejs/security-wg/blob/master/processes/security_team_members.md#team-that-triages-security-reports-against-node-core)
+* Discuss issue linked with potential confusion between 2 HackerOne orgs from security reporters
+* @vdeturckheim to create email alias for 3rd party vulnerability report
+* @vdeturckheim to open an issue to open an issue to collect inputs regarding early disclosure. 
+* @mgalexander prepare proposal for bug bounty
+* @bengl agreed to socialize the discussion around security team membership (need to confirm)
```

Comment:
This isn't quite what I thought I agreed to. I'll re-listen to the audio and get back to you with a better wording of this.

Reply:
Can correct the notes to what you intended -- I totally did not mean to sign you up for something other than you were trying to step up to do. I just did not capture it in the moment. I'll amend if you want to just comment it here... thank you for looking!

Reply:
`* @bengl agreed to PR security team membership policy` is probably most accurate.

Reply:
Thank you